.github/workflows/tests.yml

```diff
@@ -24,10 +24,10 @@ on:
 jobs:
   python-tests:
     strategy:
-      max-parallel: 9
+      max-parallel: 3
       matrix:
-        os: [ubuntu-latest, macos-latest]
-        python-version: [3.7, 3.8, 3.9]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.9]
```

Comment:
Maybe we could also have a linting check run first like PySyft and then run other runners if it passes , so we could use our runners optimally.

Reply:
https://github.com/OpenMined/PySyft/blob/563d00f22c7577fcd616b43397737eac09ed6204/.github/workflows/syft-pr_tests.yml#L25